src/App.tsx

```diff
@@ -1,18 +1,26 @@
-import * as React from 'react';
-import { ChakraProvider, Box, theme } from '@chakra-ui/react';
-import Header from './components/Layout/Header';
+import * as React from "react";
+import { ChakraProvider, Box, theme } from "@chakra-ui/react";
+import Header from "./components/Layout/Header";
 
-import './App.css';
-import Footer from './components/Layout/Footer';
-import { BrowserRouter as Router } from 'react-router-dom';
-import AppRoutes from './AppRoutes';
+import "./App.css";
```

Comment:
Put this after the react-toastify CSS import to avoid the cascading of styles in App.css.